awscli/examples/rum/batch-delete-rum-metric-definitions.rst

```diff
@@ -0,0 +1,11 @@
+**To remove the specified metrics from being sent to an extended metrics destination**
+
+The following ``batch-delete-rum-metric-definitions`` removes the specified metrics from being sent to an extended metrics destination. If the command succeeds, no output is returned. ::
+
+	aws rum batch-delete-rum-metric-definitions \
```

Comment:
All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general